package.json

```diff
@@ -71,6 +71,9 @@
       "pre-commit": "lint-staged"
     }
   },
+  "dependencies": {
+    "playwright": "1.12.2"
+  },
```

Comment:
If this is an actual dependency that you want packaged in the cypress binary, you have to add it to the `package.json` of `packages/launcher` and `packages/server`.

Reply:
Yeah, my idea was to make it an optional peer dependency so when the user installs it Cypress will do its thing? To avoid Cypress needs to bundle with `playwright`-packages or do you prefer your approach?

Reply:
I am resolving this issue for now until more feedback on the PR itself has been received. Happy to discuss this further at a later time